### PR TITLE
Custom Widget: Covid By Location

### PR DESCRIPTION
**Pull request by elenagiere**
This widget includes a chart and table with covid statistics based on county selected.

### CODE REVIEW

File: Examples/Widgets/CovidByLocation/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@customElement("covid-by-location")
+export default class CovidByLocation extends LitElement {
+  @internalProperty() private contacts: string[] = [];
+  @property({ type: String }) selectedCountyState = "Santa Clara County, CA";
+  /**
+   * Property: apiKey
+   * Access API Key: Covid Act Now Website
+   * https://apidocs.covidactnow.org/access
+   */
+  @property({ type: String, reflect: true, attribute: "api-key" }) apiKey = "1318b408f01c4aa3b5f79dedc6c90848";
```
Comment (yananym) [inline Examples/Widgets/CovidByLocation/src/index.ts:23]: we should remove this api key, right?

File: lit-element/package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
         "test": "jest"
     },
     "dependencies": {
-        "@agentx/agentx-js-api": "^1.1.3"
+        "@agentx/agentx-js-api": "^1.2.0"
```
Comment (yananym) [inline lit-element/package.json:25]: wonder why these files show as changed... do you need to update this fork?

File: Examples/Widgets/CovidByLocation/src/index.ts (hunk 1 above)
Comment (elenagiere) [reply Examples/Widgets/CovidByLocation/src/index.ts:23]: haha yes, I keep removing, but then putting it back to test :) I'll remove it.

File: lit-element/package.json (hunk 2 above)
Comment (elenagiere) [reply lit-element/package.json:25]: I can do so

Comment (elenagiere) [reply lit-element/package.json]: DONE

**Review by yananym — APPROVED** [review]

**Pull request merged**